ci-operator/populate-secrets-from-bitwarden.sh

```diff
@@ -16,6 +16,25 @@ set -o xtrace
 #          content of the BitWarden vault after your current session was started,
 #          you will need to create a new session to be able to view those changes.
 
+readonly CURRENT_OC_CONTEXT=$(oc config current-context)
```

Comment:
This shows the name of the context, which can be changed and might not be in this format.

Reply:
The comment was more about a false positive case than a false negative. Either way, being 100% correct is probably not worth it here.

Reply:
Do not know other commands for getting those parameters.
We could change accordingly if the output format is changed in the future.